### PR TITLE
Replace extensions apiGroup/apiVersion because of deprecation

### PR DESCRIPTION
**Pull request by czunker**
Deprecation Info:
https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

Signed-off-by: Christian Zunker <christian.zunker@codecentric.cloud>

**What type of PR is this?**

/kind bug

/kind chart-release

**Any specific area of the project related to this PR?**

/area falco-chart
/area falco-exporter-chart
/area falcosidekick

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Checklist**
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] CHANGELOG.md updated

### CODE REVIEW

File: falco-exporter/templates/role.yaml
Comment (leogr) [inline falco-exporter/templates/role.yaml:15]: With this, we need to bump the falco-exporter version too.

Comment (czunker) [reply falco-exporter/templates/role.yaml:15]: Thanks, missed that.

**Review by leogr — CHANGES_REQUESTED** [review]
Just a quick look.

This PR also modifies falco-exporter, so we have to bump its version too and update the changelog.

**Comment by czunker** [comment]
@leogr Thanks for the review.
I pushed changes to the falco-exporter CHANGELOG and bumped the chart version.

**Review by leogr — COMMENTED** [review]
Thank you for the quick fix.

Seems good to me now, I just have to try it before the final approval.

/cc @nibalizer 
/cc @Issif

**Review by Issif — APPROVED** [review]

**Comment by Issif** [comment]
Totally good for me. Thanks @czunker

**Review by leogr — APPROVED** [review]
Thank you!

**Pull request merged**
